config/features/config.go

```diff
@@ -234,8 +238,22 @@ func ConfigureBeaconChain(ctx *cli.Context) error {
 		cfg.DisablePullTips = true
 	}
 	if ctx.Bool(enableVecHTR.Name) {
-		logEnabled(enableVecHTR)
-		cfg.EnableVectorizedHTR = true
+		sigc := make(chan os.Signal, 1)
+		signal.Notify(sigc, syscall.SIGILL)
+		defer signal.Stop(sigc)
+		buffer := make([][32]byte, 2)
+		err := gohashtree.Hash(buffer, buffer)
+		if err != nil {
+			log.Error("could not test if gohashtree is supported")
```

Comment:
Is there a path where this error occurs for some reason, and so the code that would trigger `SIGILL` doesn't run, then we go through the timeout path and `cfg.EnableVectorizedHTR` is set to true? maybe break this whole bit out into a separate function so you can early return, or make the `select` conditional on `err == nil`.

Reply:
Nope, the only error from gohashtree is bounds check. Which are satisfied here

Reply:
But yeah, an else on the select makes a  lot of sense